dashboard/cron.py

```diff
@@ -492,7 +492,7 @@ def submission(self):
             submission_url = f"""with sub_fact as (select submission_id, assignment_id, course_id, user_id, global_canvas_id, published_score from submission_fact sf join user_dim u on sf.user_id = u.id where course_id = '{data_warehouse_course_id}'),
                 enrollment as (select  distinct(user_id) from enrollment_dim where course_id = '{data_warehouse_course_id}' and workflow_state='active' and type = 'StudentEnrollment'),
                 sub_with_enroll as (select sf.* from sub_fact sf join enrollment e on e.user_id = sf.user_id),
-                submission_time as (select sd.id, sd.submitted_at, sd.graded_at, sd.posted_at at time zone 'utc' at time zone '{settings.TIME_ZONE}' as grade_posted_local_date from submission_dim sd join sub_fact suf on sd.id=suf.submission_id where posted_at is not null and posted_at < getdate()),
+                submission_time as (select sd.id, sd.submitted_at, sd.graded_at, sd.posted_at at time zone 'utc' at time zone '{settings.TIME_ZONE}' as grade_posted_local_date from submission_dim sd join sub_fact suf on sd.id=suf.submission_id where (posted_at is not null and posted_at < getdate()) OR (posted_at is null and grade is null)),
```

Comment:
I tried to manually create a case for [submitted, graded and not posted] and I see the UI is reflecting the score 

![Screen Shot 2021-01-11 at 8 46 33 AM](https://user-images.githubusercontent.com/8579775/104190099-90fe4380-53e9-11eb-89a5-619f1e27bf5f.png)
![Screen Shot 2021-01-11 at 8 47 13 AM](https://user-images.githubusercontent.com/8579775/104190182-b428f300-53e9-11eb-97ef-ed3b81425311.png)

Reply:
Things are working as expected in the all below cases

1 - Submitted, graded, and posted
2 - Submitted, graded, and NOT posted
3 - Submitted, not graded, and not posted
4 - Not submitted, not graded, not posted